oic.wk.nmon.raml

```diff
@@ -0,0 +1,80 @@
+#%RAML 0.8
+title: Maintenance
+version: v1-20171220
+documentation:
+
+  - title: OCF Copyright
+    content: Copyright (c) 2018 Open Connectivity Foundation, Inc. All rights reserved.
+  - title: OCF License
+    content: !include LICENSE.md
+  - title: OCF Disclaimer
+    content: !include DISCLAIMER.md
+
+schemas:
+  - NMON: !include schemas/oic.wk.nmon-schema.json
+
+traits:
+  - interface-all:
+      queryParameters:
+        if:
+          enum: ["oic.if.rw","oic.if.baseline"]
+  - interface-rw:
+      queryParameters:
+        if:
+          enum: ["oic.if.rw"]
+
+/nmonResURI:
+  displayName: Network Monitoring
+  description: |
+    The resource through which a Device can monitor network traffic.
+
+  get:
+    is: [ interface-all ]
+    description: Retrieve the network monitor action status
+    responses:
+      200:
+        body:
+          application/json:
+            schema: NMON
+            example: |
+              {
+                "rt":   ["oic.wk.nmon"],
+                "ianaifType":   71,
+                "reset":   false,
+                "col" : false,
+                "time" : "2018-01-30T19:43:37Z",
+                "tx" : 10,
+                "rx" : 15,
+                "mms" : 50,
+                "ams" : 35
+              }
+
+  post:
+    is: [ interface-rw ]
+    description: |
+      Start/Stop collecting and reset the networking monitor resource
+    body:
+      application/json:
+        schema: NMON
+        example: |
+          {
+            "col":   true,
+            "reset":   true
+          }
+    responses:
+      200:
+        body:
+          application/json:
+            schema: NMON
+            example: |
+              {
+                "rt":   ["oic.wk.nmon"],
+                "ianaifType":   71,
+                "reset":   false,
+                "col" : true,
+                "time" : "",
```

Comment:
Not sure what happened to my comment but this needs to be updated to reflect a valid date-time.